omnibus/config/projects/chef.rb

```diff
@@ -20,7 +20,7 @@
 homepage "https://www.chef.io"
 
 build_iteration 1
-build_version "12.6.0"
+build_version "12.7.0"
```

Comment:
Any idea how we're planning to do things when Ci owns the "patch" version? Probably something in the build will have to modify a file in the git repo and run the rake task?

On the other hand whatever you do probably won't be a lot of work so if we end up having to redo it to make the Ci-auto-patch version thing work, there won't be a lot of wasted effort.